src/main/kotlin/org/rust/lang/core/psi/ext/RsPat.kt

```diff
@@ -18,3 +18,23 @@ val RsPat.isIrrefutable: Boolean
         is RsPatConst, is RsPatRange -> false
         else -> true
     }
+
+val RsPat.isRefutable: Boolean
```

Comment:
Master already contains `isIrrefutable` property, so maybe there is no need for this code?

Reply:
Fixed.